.markdownlint.yaml

```diff
@@ -0,0 +1,40 @@
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
+default: true
+
+# Hard tabs
+MD010:
+  spaces_per_tab: 4
+
+# Line length
+MD013:
+  code_blocks: false
+  headings: false
```

Comment:
Nit: do we not want to apply this rule to code_blocks and headings?

Reply:
just enabled